gnd/build.gradle

```diff
@@ -115,6 +116,7 @@ dependencies {
     implementation "com.google.firebase:firebase-core:$project.firebaseCoreVersion"
     implementation "com.google.firebase:firebase-firestore:$project.firebaseFirestoreVersion"
     implementation "com.google.firebase:firebase-auth:$project.firebaseCoreVersion"
+    implementation "com.google.firebase:firebase-perf:16.0.+"
```

Comment:
Please use `$project.firebaseCoreVersion` for version number instead.

Reply:
There's a compilation error when I try to use `firebaseCoreVersion` for the perf dependency.

Moreover, Firebase Android SDKs now have independent version numbers.
https://stackoverflow.com/q/50342230
https://firebase.google.com/support/release-notes/android

Reply:
Ah, I see. In that case please define a variable with the version number like we do for other deps (`$project.firebasePerf`) and define a deterministic version number (instead of `.+`).

Reply:
Done